File: WebApplication/Models/ClientCommands.cs
using System.Collections.Generic;

namespace CocaineCartels.WebApplication.Models
{
    public class ClientCommands
    {
        public IEnumerable<ClientAllianceProposal> AllianceProposals { get; set; }
        public IEnumerable<ClientMoveCommand> MoveCommands { get; set; }
        public IEnumerable<ClientPlaceCommand> PlaceCommands { get; set; }
    }
}

File: WebApplication/Client/ClientCommands.ts
module CocaineCartels {
    "use strict";

    export class ClientCommands {
        constructor(
            public allianceProposals: Array<ClientAllianceProposal>,
            public moveCommands: Array<ClientMoveCommand>,
            public placeCommands: Array<ClientPlaceCommand>
        ) { }
    }
}
